chihacknight/il-solar-map
# update data for dec 2024

## Overview

Data update for Dec 2024

## Testing Instructions

* electrified and planned numbers should be consistent in text and table
* confirm update post shows as expected
